### DISCUSSION TITLE
[Feature request]:Can I install TOPAS on Windows?

### DISCUSSION THREAD

**Discussion by omarRgg** [OpenTOPAS/OpenTOPAS#4] [Feature-Request] [answered]
### Feature request

Is it possible to install TOPAS in Windows, and is there any instruction available for this?

### TOPAS category

Outcome model

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

### Attestation

- [X] I testify that I have fully read the bug reporting guideline beforehand.

**Comment by JoseRamosMendez** [comment, marked as answer]
TBD

**Comment by yezr1905** [comment]
@omarRgg @JoseRamosMendez 
Yes, it is possible to install TOPAS on Windows platforms, although the official versions typically focus on Unix-based systems. Recently, I've added support for Geant4 v11.2.1 and Windows compatibility to TOPAS v3.9. [yezr1905/topas_yezr](https://github.com/yezr1905/topas_yezr) But I did not know about OpenTOPAS project when I did that.

TOPAS is a powerful framework that has greatly aided my research in medical physics. I am happy to find OpenTOPAS. As a PhD student, my expertise in C++ and CMake is not profound. Please note that there may be some minor bugs due to these additions in my code.

**Comment by JNaokiDKondo** [comment]
The OpenTOPAS ReadTheDocs has been updated to include a QuickStart guide for the Windows Subsystem for Linux (WSL) found at https://opentopas.readthedocs.io/en/latest/getting-started/WSL.html
